pyethapp/app.py

```diff
@@ -90,11 +91,16 @@ def app(ctx, profile, alt_config, config_values, data_dir, log_config, bootstrap
     # config files only contain required config (privkeys) and config different from the default
     if alt_config:  # specified config file
         config = konfig.load_config(alt_config)
+        if not config:
+            log.warning('empty config given')
```

Comment:
does it work w/ at all with empty config? what would happen? can you please document it.